ambry-router/src/main/java/com/github/ambry/router/PutBlobMetaInfo.java

```diff
@@ -132,8 +164,15 @@ public static PutBlobMetaInfo deserialize(String metaInfo) throws IOException {
       throw new IOException("Not expected JSON content " + metaInfo + e.getMessage());
     }
 
+    // version id
+    JsonNode jsonNode = rootNode.get(VERSION);
+    short version = jsonNode.shortValue();
+    if (version != VERSION_1) {
+      throw new IOException("Wrong version number " + metaInfo);
+    }
+
```

Comment:
for json serialization and deserialization, why are we using both jackson and org.json at the same time? We only need one library to deal with JSON. I suppose we just use jackson and remove org.json.

Reply:
Changed to Jackson. Thanks for the suggestion.